pages/user/[id].vue

```diff
@@ -304,8 +414,10 @@ if (user.value.username !== route.params.id) {
 
 const metaDescription = ref(
   user.value.bio
-    ? `${user.value.bio} - Download ${user.value.username}'s projects on Modrinth`
-    : `Download ${user.value.username}'s projects on Modrinth`
+    ? `${user.value.bio} - ${formatMessage(messages.profileMetaDescription, {
+        username: user.value.username,
+      })}`
+    : `${formatMessage(messages.profileMetaDescription, { username: user.value.username })}`
```

Comment:
I think since we're forming the text here, having two separate strings where one includes bio and other doesn't makes a lot more sense. I'm not sure other languages would use a dash (—) as separator, so yeah.